Validation/RecoVertex/interface/PrimaryVertexAnalyzer4PUSlimmed.h

```diff
@@ -120,7 +120,7 @@ class PrimaryVertexAnalyzer4PUSlimmed : public DQMEDAnalyzer {
     int kind_of_vertex;
     std::vector<const TrackingVertex *> sim_vertices;
     std::vector<const simPrimaryVertex *> sim_vertices_internal;
-    std::vector<unsigned int> sim_vertices_num_shared_tracks;
+    std::vector<float> sim_vertices_num_shared_tracks;
```

Comment:
Picky comment, but does the variable name make sense, now that the type was changed to `float`?

Reply:
Yes, that would be great, but only if it's not too much of a hassle!